cmd/completion/main.go

```diff
@@ -16,13 +22,18 @@ const (
 	registrySecretsDir = "/var/build-secrets"
 	reportFilePath     = "/var/report/report.toml"
 	notarySecretDir    = "/var/notary/v1"
+	buildNumberKey     = "buildNumber"
```

Comment:
are these 2 `const`s used?

Reply:
We are currently using those for annotating cosign signatures so that each signature contains the `buildNumber` and `buildTimestamp`

Reply:
hm, they are still showing as unused in my editor

Reply:
OHHH this is the one in completion/main. My bad, I thought it was in build_pod... And yes, weird, VScode normally shows unused but you are right, I am no longer using those values. Will delete